src/Twilio/Version.php

```diff
@@ -83,7 +83,9 @@ protected function exception(Response $response, string $header): TwilioExceptio
         if (\is_array($content)) {
             $message .= isset($content['message']) ? ': ' . $content['message'] : '';
             $code = isset($content['code']) ? $content['code'] : $response->getStatusCode();
-            return new RestException($message, $code, $response->getStatusCode());
+            $details = isset($content['details']) ? $content['details'] : $response->getDetails();
```

Comment:
The response object doesn't have a `getDetails` method.

Reply:
removed